src/core/Discord/HttpService.cpp

```diff
@@ -6,6 +6,9 @@
 #include <QtCore/QBuffer>
 #include <QtCore/QJsonDocument>
 #include <QtNetwork/QNetworkReply>
+#include <QtNetwork/qhttpmultipart.h>
+#include <QtGui/QPixmap>
+#include <QtCore/qfile.h>
```

Comment:
Prefer using the "extension-less" includes! so QHttpMultiPart and QFile respectively here

Reply:
ok